pom.xml

```diff
@@ -1602,7 +1606,7 @@
     <profile>
       <id>mr</id>
       <modules>
-        <module>client-mr</module>
+        <module>client-mr/core</module>
```

Comment:
Could we move module to here?
```
<module>client-mr/hadoop${hadoop.short.version}-shim</module>
```
When we enable hadoop, we can only change modify the hadoop version.